Helper/FeatureSwitches/Config.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Helper\FeatureSwitches;
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+
+class Config extends AbstractHelper {
+    const TABLE_NAME = 'bolt_feature_switches';
+
+    const SWITCH_NAME_COL = 'switch_name';
```

Comment:
I'm not super familiar with magento2's ORM but I wonder if we can define php object for this table and utilize magento's repository for CRUD operations? @danac-gs @matt-thomason

Reply:
i'm going to come back to creating a model in a new PR. this pr jsut creates the migration.